CONTRIBUTING.md

```diff
@@ -0,0 +1,81 @@
+# How to Contribute
+
+Thank you for taking the time to contribute! 
+
+We're so thankful you're considering contributing to an [open source project of the U.S. government](https://code.gov/)! If you're unsure about anything, please feel free to reach out to the points of contact documented in the project readme. The worst that can happen is you'll be politely asked to change something. We appreciate all friendly contributions.
+
+We encourage you to read this project's [LICENSE](LICENSE.md), [CODE_OF_CONDUCT](CODE_OF_CONDUCT.md), and its [README](README.md).
+
+## How Can I Contribute?
+
+There are a number of ways to contribute to this project.
+
+### Report a Bug
+
+If you do not find your bug in our issues list, file a bug report. When reporting the bug, please follow these guidelines:
+
+- **Please use the [Bug Report](https://github.com/HHS/TANF-app/issues/new?assignees=&labels=dev&projects=&template=bug-template.md&title=) issue template** This is populated with information and questions that will help TANF Data Portal (TDP) developers resolve the issue
+- **Use a clear and descriptive issue title** for the issue to identify the problem.
+- **Describe the exact steps to reproduce the problem** in as much detail as possible. For example, start by explaining how you got to the page where you encountered the bug and what you were attempting to do when the bug occurred.
+- **Describe the behavior you observed after following the steps** and point out what exactly is the problem with that behavior.
+- **Explain which behavior you expected to see instead and why.**
+- **Include screenshots, animated GIFs, or videos** if possible, which show you following the described steps and clearly demonstrate the problem.
+- **If the problem wasn't triggered by a specific action**, describe what you were doing before the problem happened.
+
+### Suggest an Enhancement
+
+If you don't have specific language or code to submit but would like to suggest a change, request a feature, or have something addressed, you can open an issue in this repository.
+
+In this issue, please describe the use case for the feature you would like to see -, what you need, why you need it, and how it should work. Team members will respond to the Feature request as soon as possible. Often, Feature request suggestions undergo a collaborative discussion with the community to help refine the need for the feature and how it can be implemented.
+
+## Code Contributions
+
+### Getting Started
+
+Please refer to readmes in the [project repo's root](https://github.com/HHS/TANF-app/blob/main/README.md), [how we work](https://github.com/HHS/TANF-app/blob/main/docs/How-We-Work/README.md), and [technical documentation](https://github.com/HHS/TANF-app/blob/main/docs/Technical-Documentation/README.md) folders for detailed information on contributing to the project.
+
+### Workflow and Branching
+
+All contributions are directed toward the `main` branch.
```

Comment:
@reitermb why this branch? are we directing people to open PRs in HHS/TANF-app:main (staging)?

Reply:
okay. i think for now, we'd encourage contributions via opening issues in HHS:main, but we'd need to revisit the topic of opening PRs in the HHS repo until we can revisit our gitflow, which currently doesnt account for external contributions. 

I'm also not sure what best practice is, but since our project's currently setup relies on most contributions originating from our vendor's fork, we'd want to consider a manageable workflow for this, including:

- how to determine if we should consider the contribution
- if the contribution is considered, how do we get the contribution connected to our vendor's repo, where our team can have more control of priority, review and testing?